docs/concepts/internationalization.md

```diff
@@ -58,11 +67,9 @@ To respect the language settings of clients, the server should have clients loca
 
 The first parameter of the `TranslatableContents(String, Object...)` constructor is a translation key, and the rest are used for formatting. The only supported format specifiers are `%s` and `%1$s`, `%2$s`, `%3$s` etc. Formatting arguments may be `Component`s that will be inserted into the resulting formatted text with all their attributes preserved.
 
-A `MutableComponent` can be created using `Component#translatable` by passing in the `TranslatableContents`'s parameters. It can also be created using `MutableComponent#create` by passing in the `ComponentContents` itself.
```

Comment:
Why is this being removed? And where is its replacement?

Reply:
```
`Component` itself provides multiple static methods for constructing `MutableComponent` - the only implementation of `Component` - with various `ComponentContents`. `MutableComponent` may also be constructed by providing `ComponentContents` directly with `MutableComponent#create(ComponentContents)`
```
I've removed it because I thought this line should cover this in more general way.

Reply:
Generalization is fine, but since this page is about translation, it needs some discussion on what this component is and how to use it. You can stick this all in a table, but I don't think it should be outright removed.